CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Removed Features
 
 ### Deprecated Fatures
+- ROX-25677: The format for specifying durations in JSON requests will soon be restricted to a simpler format.
```

Comment:
```suggestion
- ROX-25677: The format for specifying durations in JSON requests will be restricted to a numeric format.
```

Reply:
Technically it's not numeric as it has `s` suffix